Document new beyla_network_inter_zone_bytes metric (#1602)
grafana/beyla: docs/sources/network/config.md

## Network metrics configuration properties

> To enable network metrics, you need to add one of the following `features` to either

Reviewer: ```suggestion
To enable network metrics add one of the following `features` to either
```

> To enable network metrics, you need to add one of the following `features` to either

Reviewer: ```suggestion
To enable network metrics add one of the following `features` to either
```

the [otel_metrics_export]({{< relref "../configure/export-data.md" >}}))
or [prometheus_export]({{< relref "../configure/export-data.md#prometheus-http-endpoint" >}}))
configuration properties:

> * `network` enables the `beyla_network_flow_bytes` metric: the number of bytes between two endpoints of your cluster.

Reviewer: ```suggestion
* `network` enables the `beyla_network_flow_bytes` metric: the number of bytes between two endpoints of your cluster
```

Reviewer: We don't use fullstops at the end of unordered lists.

* `network_inter_zone` enables `beyla_network_inter_zone_bytes` metric: the number of bytes between different

>   availability zones in your Cloud cluster.

Reviewer: ```suggestion
  availability zones in your Cloud cluster
```

{{< admonition type="caution" >}}
The `beyla_network_inter_zone_bytes` specification is currently in experimental and only available for Kubernetes cluster.

> You might expect some breaking changes in future versions of Beyla.

Reviewer: ```suggestion
The specification is not final and future version of Beyla may introduce breaking changes.
```
